yarn-project/simulator/src/public/public_tx_simulator.test.ts

```diff
@@ -510,12 +541,14 @@ describe('public_tx_simulator', () => {
 
     // we keep the non-revertible data.
     expect(countAccumulatedItems(output.accumulatedData.nullifiers)).toBe(4);
-    expect(output.accumulatedData.nullifiers.slice(0, 4)).toEqual([
+    const siloedNullifiers = [
       new Fr(7777),
       new Fr(8888),
       siloNullifier(contractAddress, new Fr(1)),
       siloNullifier(contractAddress, new Fr(5)),
-    ]);
+    ];
+    expect(output.accumulatedData.nullifiers.slice(0, 4)).toEqual(siloedNullifiers);
+    await checkNullifierRoot(siloedNullifiers, txResult);
```

Comment:
We can check the root this way once we have alignment in how insertions will happen in the AVM & base rollup etc